playground/infrastructure/cloudbuild/cloudbuild_pg_infra.yaml

```diff
@@ -0,0 +1,53 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+steps:
+  # This step builds the image from Dockerfile that will be used as Docker machine for Beam Playground deployment
+  - name: 'gcr.io/cloud-builders/docker'
+    args:
+      - 'build'
+      - '--tag=$LOCATION-docker.pkg.dev/$PROJECT_ID/$_ARTIFACT_REGISTRY_REPOSITORY_ID/builder:worker'
+      - '-f'
+      - 'playground/infrastructure/cloudbuild/Dockerfile-base-image-worker'
+      - '.'
+
+  # This step uses Docker container from image in previous step to run gradle task for Playground Infrastructure deployment
+  - name: '$LOCATION-docker.pkg.dev/$PROJECT_ID/$_ARTIFACT_REGISTRY_REPOSITORY_ID/builder:worker'
+    entrypoint: "/bin/bash"
+  # Please set variables $_ENVIRONMENT_NAME and $_DNS_NAME in "substitutions" stage below
+    args: ['-c', './gradlew playground:terraform:InitInfrastructure -Pproject_environment="$_ENVIRONMENT_NAME" -Pdns-name="$_DNS_NAME"']
+
+substitutions:
+  # The name of your Google Cloud Artifact Registry
+  _ARTIFACT_REGISTRY_REPOSITORY_ID: "CHANGE_ME"
```

Comment:
Can we replace these CHANGE_ME values via command line? (e.g. grep, sed)

Reply:
Open question. Researching